### PR TITLE
[CSB-2058] Remove twitter-salesforce replacing with amqp-salesforce integration

### PR DESCRIPTION
**Pull request by smongiar**

### CODE REVIEW

File: amqp-salesforce/src/main/java/org/apache/camel/example/mention/AmqpSalesforceRoute.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.example.mention;
+
+import java.lang.Class;
+import java.lang.reflect.Method;
+//import org.apache.camel.salesforce.draftdto.Contact;
```
Comment (mcarlett) [inline amqp-salesforce/src/main/java/org/apache/camel/example/mention/AmqpSalesforceRoute.java]: typo

File: amqp-salesforce/src/main/java/org/apache/camel/example/mention/AmqpSalesforceRoute.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                            jsonParser.nextToken();
+                            screenName = jsonParser.getValueAsString();
+                        }
+                    }
+                    Class contact = null;
+                    if (Class.forName("org.apache.camel.salesforce.dto.Contact") != null) {
+                        contact = Class.forName("org.apache.camel.salesforce.dto.Contact");
+                    } else {
+                        //contact = Contact.class;
+                    }
```
Comment (mcarlett) [inline amqp-salesforce/src/main/java/org/apache/camel/example/mention/AmqpSalesforceRoute.java]: typo

File: amqp-salesforce/pom.xml [hunk 3]
```diff
@@ -60,14 +60,22 @@
             <groupId>org.apache.camel.springboot</groupId>
             <artifactId>camel-spring-boot-starter</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.apache.camel.springboot</groupId>
-            <artifactId>camel-twitter-starter</artifactId>
-        </dependency>
+		<dependency>
```
Comment (Croway) [inline amqp-salesforce/pom.xml]: is format odd?

**Comment by Croway** [comment]
@smongiar I am wondering, is there any reason why you are using AMQP instead of JMS?

**Comment by Croway** [comment]
May you provide steps to install artemis, install and run the example on OCP too?

something like

https://github.com/jboss-fuse/camel-spring-boot-examples/blob/camel-spring-boot-examples-4.0.0-M3-branch/amqp/readme.adoc#trying-out-the-example-on-openshift

**Comment by smongiar** [comment]
> @smongiar I am wondering, is there any reason why you are using AMQP instead of JMS?

No reason, it's just for give demo on two supported integrations

**Review by Croway — APPROVED** [review]

**Pull request merged**